### DISCUSSION TITLE
Search for "sold by: Amazon" in the offers page not working

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#2339] [Help]
I've tried using the inStock container:

```
		inStock: {
			container: '#aod-offer-soldBy',
			text: ['Amazon', 'Amazon Warehouse', 'Amazon.com',]
```
			
 which works for amazon-uk but for some reason is not working for amazon US, still shows as in stock from 3rd party sellers not amazon only.. Anyone know why?

![search for sold by amazon](https://user-images.githubusercontent.com/78455265/114277175-01852280-9a22-11eb-9312-c28039c831a7.PNG)

**Comment by csoyuncu** [comment]
i wonder this..

**Reply by Jwoods2019** [reply]
i have been trying to get the amazon warehouse stock to show as well. Seems as only the first add to cart is being detected after the buybox is triggered. Anybody have any input on how to creect this thanks.

**Reply by lozbinks** [reply]
It seems to be working for me now however the bot is scanning really really slowly compared to the speed it scans all other amazon such as UK etc, anyone know why this could be?

**Reply by AurumLuna** [reply]
Hey
i also have been trying same thing but wont work for me. Can you tell me whats wrong with my file please. Thank you.



```
import {Store} from './store';

export const AmazonUk: Store = {
  backoffStatusCodes: [403, 429, 503],
  currency: '£',
  labels: {
    captcha: {
      container: 'body',
      text: ['enter the characters you see below'],
    },
    captchaHandler: {
      challenge: '.a-row > img',
      input: '#captchacharacters',
      submit: 'button[type="submit"]',
    },
    inStock: [
      {
        container: '#tabular-buybox',
        text: ['Amazon', 'Amazon Germany', 'Amazon.com', 'Amazon Warehouse']
      },
      {
        container: '#availability',
        text: ['in stock'],
      },
    ],
    maxPrice: {
      container: '#priceblock_ourprice',
    },
    outOfStock: [
      {
        container: '#availability',
        text: ['out of stock', 'unavailable'],
      },
      {
        container: '#backInStock',
        text: ['unavailable'],
      },
    ],
  },
```

**Reply by lozbinks** [reply]
inStock: {
      container: '#sellerProfileTriggerId',
      text: ['Amazon.co.uk'],
    },

**Reply by AurumLuna** [reply]
> ```
> inStock: {
>   container: '#sellerProfileTriggerId',
>   text: ['Amazon.co.uk'],
> },
> ```

Thank you so much. I will try now.
